### ISSUE TITLE
NVDA is not reading the headings of the forms

### ISSUE THREAD

**Issue by WiproCRMAccessibility** [nvaccess/nvda#9312]
### Steps to reproduce:
Open attached HTML. Start NVDA and then follow these steps.
Navigate through the controls in page and observe that headings of the elements are not read even when there is aria-label added or not on the respective parent div. 

### Actual behavior:
NVDA is not reading headings of the elements

### Expected behavior:
NVDA is not reading headings of the elements
[Bug1294917_NVDA_Standalone.zip](https://github.com/nvaccess/nvda/files/2894258/Bug1294917_NVDA_Standalone.zip)

**Comment by DrSooom** [comment]
@WiproCRMAccessibility: Your mentioned expected behaviour is equal to your mentioned actual one. Furthermore please completely fill out the bug report draft. Thanks.

**Comment by Adriani90** [comment]
I think I can reproduce this in Firefox 65.0.1. But only on the edit fields for name. The combo boxes are read properly. But on the edit fields, NVDA does not announce the label "name" when using e in browse mode or when using tab and shift+tab in focus mode. cc: @jcsteh

**Comment by Adriani90** [comment]
This is also reproducible in Chrome 71

**Comment by jcsteh** [comment]
Invalid test case. The divs have no role (implicit or explicit), so NVDA doesn't see them as relevant containers. There's no good reason not to use aria-labelledby directly on the field. if you must have a container, use role="group".

**Comment by WiproCRMAccessibility** [comment]
@jcsteh Thanks for suggesting to ad role as group.
[NVDA_Observations.zip](https://github.com/nvaccess/nvda/files/2918253/NVDA_Observations.zip)

Even though if I had role of section as group, when there are child controls with **ul** role as **menubar**  and **li** with role **menuitem**, then again NVDA is not reading the section heading.

Attached is the markup.

**Comment by WiproCRMAccessibility** [comment]
Raised new issue https://github.com/nvaccess/nvda/issues/9336 as this is closed.
